website/docs/ocl.md

```diff
@@ -120,25 +120,43 @@ This is the list of security (groups) defined in the service network. Each secur
     "meta": "data",
     "other": true
   },
-  "artifacts": [
-    {
-      "name": "my-artifact",
-      "type": "jar",
-      "url": "mvn:groupId/artifact/1.6",
-      "properties": {
-        "additional": "property",
-        "another": "one"
+  "image": {
+    "provisioners": [{
+      "name": "my-kafka-release",
+      "type": "shell",
+      "environment_vars": [
+        "WORK_HOME=/usr1/KAFKA/"
+      ],
+      "inline": [
+        "cd ${WORK_HOME} && wget http://xxxx/kafka/release.jar"
+      ]
+    }
+    ],
+    "base": {
```

Comment:
suggest to keep `base` and `artifacts` to List as others for now.

Reply:
Agree: a service can use several image/artifacts (as a composition). That's why I used `List<Artifact>`.

Reply:
> What this PR does / why we need it:
> 
> * Update OCL with vm/image/artifacts.
> * Throw out a disscution about the block attachment (Map or List).
> 
> Which issue this PR fixes: Resolves #10

Yes, As I mentioned in the PR header, we had a offline disscution yet. 
Now we decided to use List instead of Map.